scripts/lib/api/updateLinks.ts

```diff
@@ -25,6 +25,26 @@ import { removePart, removePrefix } from "../stringUtils";
 import { HtmlToMdResultWithUrl } from "./HtmlToMdResult";
```

Comment:
I'm curious if this PR's changes impact any of the file `ignores.ts`? If so, it might be worth having a dedicated PR to add `ConvertMarkdownAnchorsToLowerCase` and update `ignores.ts`.

Reply:
Good call, but sadly it doesn't seem to address any of those. There are some all-uppercase anchors in that file, but it seems the issue is that they're just marked as bold rather than actual headings.